beamer/models/request.py

```diff
@@ -31,7 +32,8 @@ def __init__(
         self.filler: Optional[ChecksumAddress] = None
         self.fill_tx: Optional[HexBytes] = None
         self.fill_id: Optional[int] = None
-        self.l1_resolution_filler = Optional[ChecksumAddress]
+        self.l1_resolution_filler: Optional[ChecksumAddress] = None
+        self.l1_resolution_tx_handle: Optional[Future] = None
```

Comment:
Do we really need the future handle on `Request`?

Reply:
> Yes, we need a way top track the tasks progress for this request.

Isn't the state machine supposed to be doing that?
This feels like it should rather be a new state, e.g. `submitted_l1_proof`, or something like that.